app/src/main/java/cm/aptoide/pt/socialMedia/SocialMediaNavigator.kt

```diff
@@ -0,0 +1,32 @@
+package cm.aptoide.pt.socialMedia
+
+import android.content.Context
+import cm.aptoide.aptoideviews.socialmedia.SocialMediaView
+import cm.aptoide.pt.R
+import cm.aptoide.pt.link.CustomTabsHelper
+import cm.aptoide.pt.themes.ThemeManager
+
+class SocialMediaNavigator(val context: Context, val themeManager: ThemeManager) {
+
+  companion object {
+    var FACEBOOK_URL = "https://pt-pt.facebook.com/aptoide/"
```

Comment:
We want this hardcoded for pt? What about other languages, I guess facebook redirects to the right language but still, can we have a generic url?